### PR TITLE
add authentication to show_action rest api

### PR DESCRIPTION
**Pull request by lingbin**

### CODE REVIEW

File: fe/src/com/baidu/palo/http/BaseAction.java [hunk 1]
```diff
@@ -87,6 +86,7 @@ public void setQeService(QeService qeService) {
     @Override
     public void handleRequest(BaseRequest request) throws Exception {
         BaseResponse response = new BaseResponse();
+        LOG.debug("receive http request. url=", request.getRequest().uri());
```
Comment (morningman) [inline fe/src/com/baidu/palo/http/BaseAction.java]: 少个 {}

Comment (imay) [inline fe/src/com/baidu/palo/http/BaseAction.java]: This log should remove

**Review by imay — CHANGES_REQUESTED** [review]

File: fe/src/com/baidu/palo/http/BaseAction.java (hunk 1 above)
Comment (lingbin) [reply fe/src/com/baidu/palo/http/BaseAction.java]: Done.

Comment (lingbin) [reply fe/src/com/baidu/palo/http/BaseAction.java]: this log is to show reqeust url to help debug.

**Review by imay — APPROVED** [review]

**Pull request merged**